src/App.tsx

```diff
@@ -29,127 +97,99 @@ export const App: React.FC = () => {
+                <div className="loader" />
+              </div>
             </div>
-          </div>
-
-          {/* This todo is in loadind state */}
-          <div className="todo">
-            <label className="todo__status-label">
-              <input type="checkbox" className="todo__status" />
-            </label>
-
-            <span className="todo__title">Todo is being saved now</span>
-            <button type="button" className="todo__remove">×</button>
-
-            {/* 'is-active' class puts this modal on top of the todo */}
-            <div className="modal overlay is-active">
-              <div className="modal-background has-background-white-ter" />
-              <div className="loader" />
-            </div>
-          </div>
+          ))}
         </section>
-
-        {/* Hide the footer if there are no todos */}
-        <footer className="todoapp__footer">
-          <span className="todo-count">
-            3 items left
-          </span>
-
-          {/* Active filter should have a 'selected' class */}
-          <nav className="filter">
-            <a href="#/" className="filter__link selected">
-              All
-            </a>
-
-            <a href="#/active" className="filter__link">
-              Active
-            </a>
-
-            <a href="#/completed" className="filter__link">
-              Completed
-            </a>
-          </nav>
-
-          {/* don't show this button if there are no completed todos */}
-          <button type="button" className="todoapp__clear-completed">
-            Clear completed
-          </button>
-        </footer>
+        {todos?.length > 0 && (
+          <footer className="todoapp__footer">
+            <span className="todo-count">{`${uncompletedTodos?.length} items left`}</span>
+
+            <nav className="filter">
+              <a
+                href="#/"
+                className={cn('filter__link', {
+                  selected: filterStatus === Status.ALL,
+                })}
+                onClick={() => setFilterStatus(Status.ALL)}
+              >
+                All
+              </a>
+
+              <a
+                href="#/active"
+                className={cn('filter__link', {
+                  selected: filterStatus === Status.ACTIVE,
+                })}
+                onClick={() => setFilterStatus(Status.ACTIVE)}
+              >
+                Active
+              </a>
+
+              <a
+                href="#/completed"
+                className={cn('filter__link', {
+                  selected: filterStatus === Status.COMPLETED,
+                })}
+                onClick={() => setFilterStatus(Status.COMPLETED)}
+              >
+                Completed
+              </a>
+            </nav>
+
+            <button
+              type="button"
+              className="todoapp__clear-completed"
+              style={{ opacity: completedTodos.length > 0 ? '1' : '0' }}
```

Comment:
it will be better to use another approach to hide button, since it is still clickable using opacity